app/js/reducers/supertypes.js

```diff
@@ -45,6 +46,15 @@ const reducer = (state = initialState, action) => {
         };
         return newState;
     }
+    case 'IMPORTABLE_TYPES': {
+        const newState = {
+            ...state,
+            importableTypes: action.importableTypesList.map((item) => {
+                return { value: item, label: item };
+            })
```

Comment:
Good to know!  Still grappling with the nuances of all this new syntax, but it's all very cool.  Thanks!